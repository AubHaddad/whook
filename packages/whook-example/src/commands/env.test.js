import initEnvCommand from './env';

describe('envCommand', () => {
  const log = jest.fn();

  beforeEach(() => {
    log.mockReset();
  });

  it('should work', async () => {
    const envCommand = await initEnvCommand({
      log,
      ENV: { NODE_ENV: 'test' },
      args: {
        _: ['env'],
        name: 'NODE_ENV',
      },
    });
    const result = await envCommand();

    expect({
      result,
      logCalls: log.mock.calls.filter(args => 'stack' !== args[0]),
    }).toMatchSnapshot();
  });
});
