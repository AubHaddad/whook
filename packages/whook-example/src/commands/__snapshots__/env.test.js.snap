// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`envCommand should work 1`] = `
Object {
  "logCalls": Array [
    Array [
      "info",
      "\\"test\\"",
    ],
  ],
  "result": undefined,
}
`;
